.vscode/launch.json

```diff
@@ -21,11 +21,12 @@
     {
       "cwd": "${workspaceFolder}/app/frontend",
       "env": {
-        "VITE_TITLE": "Common Hosted Forms - Local",
+        "VITE_CHEFS_ADVANCE_GEO_ADDRESS_APIURL": "http://localhost:8080/app/api/v1/bcgeoaddress/address",
```

Comment:
The `VITE_CHEFS_ADVANCE_GEO_ADDRESS_APIURL` is the only addition, the rest is just alphabetizing the variables.